setup/jest.config.js

```diff
@@ -132,7 +132,6 @@ module.exports = {
     'src/modules/common/components/timestamp/index.js',
```

Comment:
Edit bookmark allows me to remove the name and to remove it for many bookmarks as well
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/0ec0a4e5-23c7-4dc0-95cb-855bcf194101)

Reply:
Fixed